tensorflow/python/autograph/impl/api_test.py

```diff
@@ -921,6 +921,16 @@ def test_fn(x, s):
     # Just check that the output is parseable Python code.
     self.assertIsNotNone(parser.parse_str(api.to_code(test_fn)))
 
+  def test_to_code_with_wrapped_function(self):
+
+    @def_function.function
+    def test_fn(x, s):
+      while tf.reduce_sum(x) > s:
+        x /= 2
+      return x
+
+    self.assertRaises(ValueError, api.to_code(test_fn))
```

Comment:
I think it's a good idea to check the message a bit as well:

```
with assertRaisesRegex(ValueError, 'try passing.*python_function'):
  api.to_code(test_fn)
```

Reply:
Okay. I will change it to just that because it is more specific.